### PR TITLE
Adding Distributed Tensorflow Examples and Organizing TF Examples folder

### PR DESCRIPTION
**Pull request by DN6**
This PR:

Adds distributed training examples for TF-1.x and TF-2.x
Moves TF-1.x and TF-2.x examples into separate directories.

### CODE REVIEW

File: tensorflow-1/comet-tf1-distributed-estimator-multiworker-mirrored-strategy.py [hunk 1]
```diff
@@ -0,0 +1,121 @@
...
+
+print("Number of available devices: ", N_REPLICAS)
+
+BUFFER_SIZE = 10000
+BATCH_SIZE = 64
+GLOBAL_BATCH_SIZE = BATCH_SIZE * N_REPLICAS
+LEARNING_RATE = 1e-5
+EPOCHS = 10
+
+experiment = comet_ml.Experiment(log_code=True)
```
Comment (dsblank) [inline tensorflow-1/comet-tf1-distributed-estimator-multiworker-mirrored-strategy.py:33]:
```suggestion
experiment = comet_ml.Experiment()
```

The default, right?

File: tensorflow-1/comet-tf1-distributed-mirrored-strategy.py [hunk 2]
```diff
@@ -0,0 +1,123 @@
...
+
+# Import TensorFlow
+import tensorflow as tf
+
+# Helper libraries
+import numpy as np
+import os
+
+PROJECT_NAME = "tf1-mirrored"
+experiment = comet_ml.Experiment(log_code=True, project_name=PROJECT_NAME)
```
Comment (dsblank) [inline tensorflow-1/comet-tf1-distributed-mirrored-strategy.py:27]: Same

File: tensorflow-1/comet-tf1-distributed-parameter-server-strategy.py [hunk 3]
```diff
@@ -0,0 +1,171 @@
...
+        cluster_resolver.cluster_spec(),
+        job_name=cluster_resolver.task_type,
+        task_index=cluster_resolver.task_id,
+        protocol=cluster_resolver.rpc_layer or "grpc",
+        start=True,
+    )
+    if cluster_resolver.task_type == "ps":
+        server.join()
+
+    experiment = comet_ml.Experiment(log_code=True, project_name=PROJECT_NAME)
```
Comment (dsblank) [inline tensorflow-1/comet-tf1-distributed-parameter-server-strategy.py:111]: Same

**Review by dsblank — CHANGES_REQUESTED** [review]

**Review by Lothiraldan — APPROVED** [review]
LGTM

**Pull request merged**